lua/entities/gmod_track_signal/cl_init.lua

```diff
@@ -59,16 +59,18 @@ function ENT:SpawnMainModels(pos,ang,LenseNum,add)
     local TLM = self.TrafficLightModels[self.LightType]
     for k,v in pairs(TLM) do
         if type(v) == "string" and not k:find("long") then
-            if IsValid(self.Models[1][add and v..add or v]) then break else
-                if TLM[k.."_long"] and LenseNum >= 7 then
-                    self.Models[1][add and v..add or v] = ClientsideModel(TLM[k.."_long"],RENDERGROUP_OPAQUE)
+			local idx = add and v..add or v
```

Comment:
В файле используются пробелы как отступы, у тебя табуляции (1)
```suggestion
            local idx = add and v..add or v
```